system/jlib/jlog.hpp

```diff
@@ -1264,6 +1264,60 @@ interface jlib_decl IContextLogger : extends IInterface
     virtual const CRuntimeStatisticCollection & queryStats() const = 0;
 };
 
+template <class INTERFACE>
```

Comment:
worth adding a comment that this INTERFACE must be a IContextLogger or derivative.